api/src/client/ApiFormationHealth.ts

```diff
@@ -140,11 +141,13 @@ const emptyBusinessUser = createEmptyUser();
 const healthCheckUser = createEmptyUserData(emptyBusinessUser);
 const currentBusiness = getCurrentBusiness(healthCheckUser);
 const currentBusinessId = currentBusiness.id;
+const taxClearanceData = emptyTaxClearanceCertificateData;
 
 const updatedBusiness = {
   ...currentBusiness,
   formationData,
   profileData,
+  taxClearanceData,
```

Comment:
@cbrunefearless do we need this? The whole point of the change I made was to prevent needing to modify this object. You shouldn't need to be touching this file.

Reply:
yes sorry, i saw what u did, i thought u were just breakign up the totality of the object.... changed